src/Core_Command.php

```diff
@@ -129,8 +129,7 @@ public function download( $args, $assoc_args ) {
 			}
 
 			WP_CLI::log( sprintf( "Creating directory '%s'.", $download_dir ) );
-			$mkdir = \WP_CLI\Utils\is_windows() ? 'mkdir %s' : 'mkdir -p %s';
-			WP_CLI::launch( Utils\esc_cmd( $mkdir, $download_dir ) );
+			mkdir( $download_dir, 0777, true /*recursive*/ );
```

Comment:
This needs to die on failure, which I believe was the behavior for `WP_CLI::launch( Utils\esc_cmd( $mkdir, $download_dir ) );`